setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+setup(
+    name='single_cell_portal',
+    version='0.1rc1',
+    description='Convenience scripts for Single Cell Portal',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='https://github.com/broadinstitute/single_cell_portal',
+    author='Single Cell Portal team',
+    author_email='scp-support@broadinstitute.zendesk.com',
+    install_requires=['pandas', 'requests', 'scp-ingest-pipeline>=1.3.7',],
```

Comment:
Having `scp-ingest-pipeline` with version constraints here in `setup.py` while completely absent in `requirements.txt` struck me as odd.  But reading through [a relevant Stack Overflow thread](https://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py) relaxed my opinion.

Package management is relatively messy in Python, so given this approach passes the build I see no functional problem.  

But _if_ we encounter funky installation problems, the lack `scp-ingest-pipeline` in `requirements.txt` would be the first thing I'd consider.